uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1012.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 trait CRM_HRCore_Upgrader_Steps_1012 {
-
+  
```

Comment:
Just curious, why did you fix whitespace issues in a different file for this PR?

Reply:
The upgrader conflicted with that in this PR and i moved the upgrader in this PR to `CRM_HRCore_Upgrader_Steps_1012 `. That is why it appeared as if i fixed the whitespace.